include/util/struct_packing.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef STRUCT_PACKING_HPP
+#define STRUCT_PACKING_HPP
+
+/*
+    Macro to pack struct/class in both environment
+    Usage:  PACK( 
+        struct MyStruct 
+        { 
+            int Property1;
+        });
+*/
+
+#ifdef _MSC_VER
+#define PACK( input ) __pragma( pack(push, 1) ) input __pragma( pack(pop) )
```

Comment:
Thanks, just pushed an update and tested it locally.  I don't have unix dev setup so have to rely on your project travis ci to check them.